packages/styling/src/interfaces/IFontStyles.ts

```diff
@@ -4,16 +4,16 @@ import { IRawStyle } from '../interfaces/index';
  * UI Fabric font set.
  */
 export interface IFontStyles {
```

Comment:
I think that seems fine, but I think you will need to update ITheme to support partial font lists.